packages/angular_devkit/build_optimizer/package.json

```diff
@@ -16,7 +16,7 @@
     "webpack-sources": "2.2.0"
   },
   "peerDependencies": {
-    "webpack": "^4.0.0 || ^5.20.0"
+    "webpack": "^4.0.0 || ^5.30.0"
```

Comment:
Should we still support V4 here?

Reply:
It technically will still work with 4 (for now).  There is some additional refactoring/cleanup that can be done to leverage newer features though and once that is in 4 support can be dropped.